.github/pull_request_template.md

```diff
@@ -1,13 +1,23 @@
-[ATTENTION. This is the *Pull Request Template*. Replace the two following lines with the indicated information and delete this one. Do not delete anything else]
-
 Closes #[issue number]
 [Description of the changes proposed in the pull request. Include steps to replicate the behavior and screenshots if UI is updated]
 
 # Review checklist
--   [ ] Terms and conditions reflect the current change
--   [ ] Contains enough appropriate tests
--   [ ] If aimed at production, writes a new summary in `whatsnew/whatsnew-pt-PT`
--   [ ] Properly adds an entry in `changelog.md` with the change
--   [ ] If PR includes UI updates/additions, its description has screenshots
--   [ ] Behavior is as expected
--   [ ] Clean, well-structured code
+
+## View Changes
+
+- [ ] Tested both in light and dark mode
+- [ ] Description has screenshots of the UI changes
+- [ ] Text is both in portuguese and english.
+
+## Documentation
+
+- [ ] Properly adds an entry in `changelog.md` with the change
+- [ ] Terms and conditions reflect the current change
+
+## Code
+
+- [ ] No uncessary helper functions
```

Comment:
I like the idea, but this needs to be more specific - what is considered unnecessary?
Also, this feels like something we could have a lint rule for

Reply:
A helper function/method is a method returning Widget. A Statefull/Stateless class should be created instead. 
Is there a way to create a lint rule for checking method with Widget return type?

Reply:
>Is there a way to create a lint rule for checking method with Widget return type?

Yhep, with [custom_lint](https://pub.dev/packages/custom_lint). It requires creating a separate package for the lint rules AFAIK, but the sky is the limit here :rocket:

Reply:
Perfect! I will create an issue and remove it from the template